### PR TITLE
fix RAPL.end() unit

### PR DESCRIPTION
**Pull request by vict0rsch**
Fixing #212

### CODE REVIEW

**Review by benoit-cty — DISMISSED** [review]
Hi Victor, why did you change from `kW` to `kw` ?
I understand that `W` is the good unit, referring to https://en.wikipedia.org/wiki/International_System_of_Units
Or is it to comply with camelcase ?

**Comment by vict0rsch** [comment]
Well initial thought is that we should have either:

```
Energy.kWh
Power.kW
```

or

```
Energy.kwh
Power.kw
```

but not, as we currently do

```
Energy.kwh
Power.kW
```

So I just thought having it all lower case would make sense a it reduces the chances of typos in the code, but I'd be fine with `kW` everywhere if you prefer that

**Review by SaboniAmine — APPROVED** [review]

File: codecarbon/core/rapl.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+from codecarbon.core.units import Energy, Power, Time
 
 
 @dataclass
 class RAPLFile:
     name: str
     path: str
     energy_reading: Energy = Energy(0)
-    power_measurement: float = 0
+    power_measurement: float = 0  # kW
```
[the remark this answers is not in the document]
Comment (Lothiraldan) [reply codecarbon/core/rapl.py:11]: I think it is a good idea, mypy would be able to check for mixing of units. If we want to go further, we can even have a custom classes that don't wrapper float and convert them to float at the last moment.

[the remark this answers is not in the document]
Comment (benoit-cty) [reply codecarbon/core/rapl.py:11]: And a class Power already exist in CodeCarbon ;)

**Comment by vict0rsch** [comment]
> Hi Victor, why did you change from `kW` to `kw` ?
> I understand that `W` is the good unit, referring to [en.wikipedia.org/wiki/International_System_of_Units](https://en.wikipedia.org/wiki/International_System_of_Units)
> Or is it to comply with camelcase ?

Last commit is ISU compliant with `kW` and `kWh`

File: codecarbon/core/rapl.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (vict0rsch) [reply codecarbon/core/rapl.py]: The latest commit should fix this

**Pull request merged**